staging/src/k8s.io/apiserver/pkg/cel/builtins/validation.go

```diff
@@ -0,0 +1,295 @@
+/*
```

Comment:
This file also only needed if for native type validation rules.

Reply:
All unused files removed.